Appraisals

```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: true
+
```

Comment:
You can now remove all the `mri_22` requirements below.

Reply:
@deivid-rodriguez Done!